tests/builder/help.rs

```diff
@@ -964,6 +999,71 @@ OPTIONS:
     ));
 }
 
+#[test]
+#[cfg(all(feature = "wrap_help"))]
+fn possible_value_wrapped_help() {
+    #[cfg(feature = "unstable-v4")]
+    static WRAPPED_HELP: &str = "test 
+
+USAGE:
+    test [OPTIONS]
+
+OPTIONS:
+    -h, --help
+            Print help information
+
+        --possible-values <possible_values>
+            Possible values:
+                - name: Long enough help message to clearly
+                          warrant wrapping
+                - second
```

Comment:
Looking over this output again, I'm wondering if we're indenting too much that it will cause us to wrap more than we should.

Maybe we should have the `-` aligned with `Possible`

Reply:
I was also considering indenting by `TAB_WIDTH - "- ".len()`
```
Possible Values:
  - name
  - second_name
```